src/black/nodes.py

```diff
@@ -742,10 +742,6 @@ def is_multiline_string(leaf: Leaf) -> bool:
     return has_triple_quotes(leaf.value) and "\n" in leaf.value
 
 
-def is_funcdef(node: Node) -> bool:
-    return node.type == syms.funcdef
-
-
 def is_function_or_class(node: Node) -> bool:
```

Comment:
This function is suspicious since many async defs don't have async_funcdef, but this can be subject for different PR